lib/install/config/webpack/shared.js

```diff
@@ -39,7 +38,6 @@ module.exports = {
     new webpack.EnvironmentPlugin(JSON.parse(JSON.stringify(env))),
     new ExtractTextPlugin(env.NODE_ENV === 'production' ? '[name]-[hash].css' : '[name].css'),
     new ManifestPlugin({
-      publicPath: output.publicPath,
```

Comment:
@gauravtiwari Can you help me with the webpack changes? I didn't see any docs on how to test this.